assets/src/edit-story/components/focusable/focusable.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import MouseTrap from 'mousetrap';
+import 'mousetrap/plugins/global-bind/mousetrap-global-bind';
+
+/**
+ * WordPress dependencies
+ */
+import { useLayoutEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useCombinedRefs from '../../utils/useCombinedRefs';
+import getValidDOMAttributes from '../../utils/getValidDOMAttributes';
+
+function Focusable( { shortcuts, hasFocus, isGlobal, element, className, role, tabindex, forwardedRef, children, ...rest } ) {
+	const ref = useRef();
+	const setRef = useCombinedRefs( ref, forwardedRef );
```

Comment:
I should also use forwardRef, yes (which allows me to use `ref={}` in the parent component), but I still need some way to merge the two refs, when needing the ref both in the component itself and in a parent component. It can't be done any other way, unfortunately.

Reply:
A single forwarded ref usually does the trick for me, but sure, if there's need, let's keep.